### PR TITLE
Kinetic Things and Weird Goggle

### PR DESCRIPTION
**Pull request by FluffMedic**
Okay
Does a funny fix/balance change? for rare kinetic weaponry
And adds an admin spawn item, mostly for discussion and expirmentation around such an effect existing.
## Changelog
:cl:
add: Added admin spawn set of goggles that's sole purpose is seeing kin (Rae: commented out the meson/kin hybrid though)
balance: Makes it so the large kinetic weapons don't need wielding since that code is appears broken and I doubt the two very rare items will become broken if they get the kinetic blast effect.
/:cl:

### CODE REVIEW

**Comment by Kazzc** [comment]
I like this a lot, may come in handy with some of my events.

**Comment by Kashargul** [comment]
Were those tested? Invisibility used here was never defined as plane, so this might runtime horribly.

File: modular_chomp/code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -6,3 +6,28 @@
...
+
+	icon = 'modular_chomp/icons/inventory/eyes/glasses.dmi'
+	icon_override = 'modular_chomp/icons/inventory/eyes/mob.dmi'
+	icon_state = "darksight"
+
+	action_button_name = "Toggle Goggles"
+
+	toggleable = 1
+	enables_planes = list(INVISIBILITY_SHADEKIN)
+
```
Comment (Kashargul) [inline modular_chomp/code/modules/clothing/glasses/glasses.dm:23]: This is not a plane.

~~enables_planes = list(INVISIBILITY_SHADEKIN)~~


see_invisible = INVISIBILITY_SHADEKIN

File: modular_chomp/code/modules/clothing/glasses/glasses.dm [hunk 2]
```diff
@@ -6,3 +6,28 @@
...
+/obj/item/clothing/glasses/meson/darksight
+	name = "optical meson scanner"
+	desc = "Used for seeing walls, floors, and stuff through anything."
+	icon_state = "meson"
+	item_state_slots = list(slot_r_hand_str = "meson", slot_l_hand_str = "meson")
+	action_button_name = "Toggle Goggles"
+	origin_tech = list(TECH_MAGNET = 2, TECH_ENGINEERING = 2)
+	toggleable = 1
+	vision_flags = SEE_TURFS
+	enables_planes = list(VIS_FULLBRIGHT, VIS_MESONS, INVISIBILITY_SHADEKIN)
```
Comment (Kashargul) [inline modular_chomp/code/modules/clothing/glasses/glasses.dm:33]: enables_planes = list(VIS_FULLBRIGHT, VIS_MESONS, ~~INVISIBILITY_SHADEKIN~~)
see_invisible = INVISIBILITY_SHADEKIN

**Review by Kashargul — CHANGES_REQUESTED** [review]

File: modular_chomp/code/modules/clothing/glasses/glasses.dm (hunk 2 above)
Comment (Kashargul) [reply modular_chomp/code/modules/clothing/glasses/glasses.dm:33]: if making it consistent, SEE_INVISIBILITY_SHADEKIN

create a new define in misc.dm with that name, value 55

**Review by Kashargul — APPROVED** [review]

**Comment by Raeschen** [comment]
not the keenest on shadekin sight, even if adminspawn

**Comment by Raeschen** [comment]
IMO, the meson/darksight hybrid shouldn't exist. The sole-shadekin sight glasses ARE ok i guess if only adminspawn, but i really want to keep an ability like this purely to a specialised thing.

**Pull request merged**